core/server/helpers/foreach.js

```diff
@@ -3,14 +3,9 @@
 //
 // Block helper designed for looping through posts
 const _ = require('lodash');
-const {logging, i18n, models, hbs} = require('./proxy');
+const {logging, i18n, hbs} = require('./proxy');
 const {Utils: hbsUtils, handlebars: {createFrame}} = hbs;
-
-function filterItemsByVisibility(items, options) {
-    const visibilityArr = models.Base.Model.parseVisibilityString(options.hash.visibility);
```

Comment:
@rishabhgrg can you expand on why this `parseVisibilityString` has disappeared, or leave a comment, so we have some context in the future when searching through commit history ;)

Reply:
@gargol Good point, updated now 😄 Basically the parsing is automatically taken care by filter now